src/DependencyInjection/Factory/ContentAdminFactory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\DependencyInjection\Factory;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\Admin\Content\StaticContentAdmin;
+
+/**
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
+class ContentAdminFactory implements AdminFactoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getKey()
+    {
+        return 'content';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addConfiguration(NodeBuilder $builder)
+    {
+        $builder->scalarNode('admin_class')->defaultValue(StaticContentAdmin::class)->end();
```

Comment:
Using the `::class` is surely better than putting the FQCN in string (it allows IDE refactorings for instance)